src/proxy.ts

```diff
@@ -21,5 +62,20 @@ export const send = (
   //
   // [Proxy with express.js](https://stackoverflow.com/q/10435407)
 
+  const { host = false, origin } = params;
+
+  const proxy = createProxyServer({
+    target,
+    changeOrigin: true,
+    cookieDomainRewrite: host
+  });
+
+  proxy.on('proxyReq', (proxyReq, ...args) => {
+    changeOrigin(proxyReq, origin);
+    handleBodyParsing(proxyReq, ...args);
+  });
+  proxy.on('proxyRes', removeSecurityFromCookie);
```

Comment:
Why do we need this?

Reply:
`Secure` flag seems to block cookie setting in localhost